umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj

```diff
@@ -393,3 +393,24 @@
   (-> c
       (update :DOI doi/migrate-doi-down-to-1-16)
       (dissoc :AssociatedDOIs)))
+
+(defmethod interface/migrate-umm-version [:collection "1.16.1" "1.16.2"]
+  [context c & _]
+  ;; Can't use the same solution as used in 1.16.2 -> 1.16.1 using update because when :UseConstraints
+  ;; is nil, it will be turned into (:Description nil :LIcenseUrl nil :LicenseText nil) which will
+  ;; fail validation. This is the same issue going from 1.9 -> 1.10 above.
+  (let [use-constraints (:UseConstraints c)
+        use-constraints (-> use-constraints
+                            (assoc :Description (get-in use-constraints [:Description :Description]))
+                            (set/rename-keys {:LicenseUrl :LicenseURL})
+                            util/remove-nils-empty-maps-seqs)]
+     (if use-constraints
+       (assoc c :UseConstraints use-constraints)
+       (dissoc c :UseConstraints))))
+
+(defmethod interface/migrate-umm-version [:collection "1.16.2" "1.16.1"]
+  [context c & _]
+  (update c :UseConstraints #(-> %
+                                 (assoc :Description {:Description (get % :Description)})
+                                 (set/rename-keys {:LicenseURL :LicenseUrl})
+                                 util/remove-nils-empty-maps-seqs)))
```

Comment:
Use the long-form `fn` notation here

https://clojuredocs.org/clojure.core/fn

Reply:
I have changed this to use the long form.